Kernel/Storage/AHCIPort.cpp

```diff
@@ -704,7 +744,7 @@ void AHCIPort::stop_fis_receiving() const
     m_port_registers.cmd = m_port_registers.cmd & 0xFFFFFFEF;
 }
 
-bool AHCIPort::initiate_sata_reset()
+bool AHCIPort::initiate_sata_reset(ScopedSpinLock<SpinLock<u8>>& scoped_locking)
```

Comment:
`scoped_locking` should probably describe the lock / what it's locking instead of the type of the lock?